pages/index.tsx

```diff
@@ -1,6 +1,7 @@
 import type { NextPage } from 'next'
 import Head from 'next/head'
 import Image from 'next/image'
+
```

Comment:
Look! Our ESLint is already working, and enforcing some code-style. In this case, the [`import/order`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) plugin is enforcing the [`newlines-between`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md#newlines-between-ignorealwaysalways-and-inside-groupsnever) rule, which enforces we keep empty lines in between different groups of imports: dependencies, local modules, relative modules, etc.